isis/src/control/objs/ControlNetVitals/ControlNetVitals.cpp

```diff
@@ -0,0 +1,245 @@
+#include "ControlNetVitals.h"
+
+#include <QList>
+
+#include "IException.h"
+#include "IString.h"
+#include "ControlNet.h"
+#include "ControlPoint.h"
+#include "ControlMeasure.h"
+
+namespace Isis {
+
+  ControlNetVitals::ControlNetVitals(ControlNet *cnet) {
+    m_controlNet = cnet;
+    validate();
+  }
+
+  ControlNetVitals::~ControlNetVitals() {
+  }
+
+  bool ControlNetVitals::hasIslands() {
+    // Replace this with graph call!!!$!@$!@$!@$#@%#@$%#@
+    return true;
+  }
+
+  int ControlNetVitals::numIslands() {
+    // replace this with graph call!#@$!#%#@%*($#)
+    return 1;
+  }
+
+  QList<QString> ControlNetVitals::getIslands() {
+    // TEMP, replace with graph
+    QList<QString> list;
+    list.append("CASSIS_01.cub");
+    return list;
+  }
+
+  int ControlNetVitals::numPoints() {
+    return m_controlNet->GetNumPoints();
+  }
+
+  // REFACTOR
+  int ControlNetVitals::numIgnoredPoints() {
+    int count = 0;
+    foreach(ControlPoint* point, m_controlNet->GetPoints()) {
+      if (point->IsIgnored()) count++;
```

Comment:
Don't use 1 line if statements. They are clean, but they obscure the fact that there's a check inside of the foreach loop.